### PR TITLE
new message: ADS-B vehicle

### PR DESCRIPTION
**Pull request by magicrub**
contains all the info, plus some additional status flags, from an ADS-B transmission but in MAVLink format.

### CODE REVIEW

**Comment by LorenzMeier** [comment]
Do you have a reference hardware this could be tested against?

**Comment by magicrub** [comment]
@LorenzMeier Yes, this is to support the uAvionix PING hardware. See http://www.uavionix.com/ for details. A pull-request is in the works for ArduPilot support where @tridge and I worked together to create an ADS-B simulator component to SITL as well.

**Comment by magicrub** [comment]
merged in by @tridge and a new pymavlink was released v1.1.63 to include it. Here is the ArduPilot PR that adds support for it: https://github.com/diydrones/ardupilot/pull/3207

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -2993,6 +3040,21 @@
             <field type="uint8_t" name="vtol_state" enum="MAV_VTOL_STATE">The VTOL state if applicable. Is set to MAV_VTOL_STATE_UNDEFINED if UAV is not in VTOL configuration.</field>
             <field type="uint8_t" name="landed_state" enum="MAV_LANDED_STATE">The landed state. Is set to MAV_LANDED_STATE_UNDEFINED if landed state is unknown.</field>
         </message>
+        <message id="246" name="ADSB_VEHICLE">
+          <description>The location and information of an ADSB vehicle</description>
+          <field type="uint32_t" name="ICAO_address">ICAO address</field>
+          <field type="float" name="lat">The reported latitude in degrees</field>
+          <field type="float" name="lon">The reported longitude in degrees</field>
+          <field type="uint8_t" name="altitude_type">Type from ADSB_ALTITUDE_TYPE enum.</field>
+          <field type="float" name="altitude" enum="ADSB_ALTITUDE_TYPE">Altitude(ASL) in meters</field>
```
Comment (vooon) [inline message_definitions/v1.0/common.xml]: @magicrub, @LorenzMeier seems that `enum=""` arg should be one line upper.

Comment (LorenzMeier) [inline message_definitions/v1.0/common.xml]: Yep, that's wrong. @magicrub Please fix ASAP.

**Comment by magicrub** [comment]
Fixed in PR https://github.com/mavlink/mavlink/pull/461

**Pull request closed without merging**